From ee944bdbf3422b4f6d0a4a2f7c7edc9b022a8bc2 Mon Sep 17 00:00:00 2001
Message-Id: <ee944bdbf3422b4f6d0a4a2f7c7edc9b022a8bc2@dist-git>
From: Peter Krempa <pkrempa@redhat.com>
Date: Tue, 2 Aug 2016 13:41:57 +0200
Subject: [PATCH] qemu: command: Rename qemuBuildNetworkDriveURI to
 qemuBuildNetworkDriveStr

The function builds also non-uri strings for the various protocols.

(cherry picked from commit b8dc04a7742c2ebabf6922ac510389945ca0404b)
https://bugzilla.redhat.com/show_bug.cgi?id=1247521 [gluster multi-host]
---
 src/qemu/qemu_command.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/qemu/qemu_command.c b/src/qemu/qemu_command.c
index 9ac90f7..b691743 100644
--- a/src/qemu/qemu_command.c
+++ b/src/qemu/qemu_command.c
@@ -691,7 +691,7 @@ qemuBuildRBDSecinfoURI(virBufferPtr buf,
 #define QEMU_DEFAULT_NBD_PORT "10809"
 
 static char *
-qemuBuildNetworkDriveURI(virStorageSourcePtr src,
+qemuBuildNetworkDriveStr(virStorageSourcePtr src,
                          qemuDomainSecretInfoPtr secinfo)
 {
     char *ret = NULL;
@@ -926,7 +926,7 @@ qemuGetDriveSourceString(virStorageSourcePtr src,
         break;
 
     case VIR_STORAGE_TYPE_NETWORK:
-        if (!(*source = qemuBuildNetworkDriveURI(src, secinfo)))
+        if (!(*source = qemuBuildNetworkDriveStr(src, secinfo)))
             goto cleanup;
         break;
 
@@ -4533,7 +4533,7 @@ qemuBuildSCSIiSCSIHostdevDrvStr(virDomainHostdevDefPtr dev)
     src.nhosts = iscsisrc->nhosts;
 
     /* Rather than pull what we think we want - use the network disk code */
-    source = qemuBuildNetworkDriveURI(&src, hostdevPriv->secinfo);
+    source = qemuBuildNetworkDriveStr(&src, hostdevPriv->secinfo);
 
     return source;
 }
-- 
2.9.2

